modules/sugar/package.json

```diff
@@ -12,7 +12,7 @@
     "build": "tsc",
     "test": "bedrock-auto -b chrome-headless --stopOnFailure -d src/test/ts",
     "test-manual": "bedrock --stopOnFailure -d src/test/ts",
-    "lint": "eslint --config ../../.eslintrc.json --max-warnings=0 src/**/*.ts"
+    "lint": "eslint --config ../../eslint.config.ts --max-warnings=0 src/**/*.ts"
```

Comment:
If the new eslint is better at inheriting from parent folders, we could start leaving this `--config` specifier off the individual module scripts 🤔

Reply:
It seems it is better at inheriting from parent folders indeed!